flumine/execution/transaction.py

```diff
@@ -224,5 +251,15 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
+        if self.all_or_nothing:
+            for order in self._pending_place:
+                self._do_validate_controls(order, OrderPackageType.PLACE)
```

Comment:
If this was to fail we would be left with orders potentially still in the `_pending` vars so it needs to be cleaned up with similar logic seen in `execute`